Assets/Scripts/GameManager.cs

```diff
@@ -24,6 +24,12 @@ public class GameManager : MonoBehaviour
     [SerializeField]
     private int level = 1;
 
+    [Header("Pickups")]
+    [SerializeField] private GameObject _hpPickup;
+    [SerializeField] private GameObject _dmgPickup;
+    [SerializeField] private GameObject _shieldPickup;
```

Comment:
Name '_hpPickup' does not match rule 'Unity serialized field'. Suggested name is 'hpPickup'.

Același lucru pentru toate 3.